metabase/metabase
# FE - e2e - Cover combination of unit of time parameter + non-last-stage filters with tests

Closes #48445

### Description

This PR:
- adds a test that caught some issues in FE-side implementation
- fixes these issues

### How to verify

The following 4 e2e tests are expected to fail due to #48787:
- `multiple-column-breakouts.cy.spec.ts`
    - `should be able to use temporal-unit parameters with multiple temporal breakouts of a column`
- `visualizations-tabular/pivot_tables.cy.spec.js`
    - `should allow filtering drill through (metabase#14632) (metabase#14465)`
- `temporal-unit-parameters.cy.spec.js`
    - `should connect a parameter to a question and drill thru`
    - `should connect multiple parameters to a card with multiple breakouts and drill thru`

In all these cases drilling works but "pivot table" setting does not get enabled/properly recognized in the table viz.
